test/cpp/end2end/xds/xds_end2end_test.cc

```diff
@@ -2031,8 +2031,6 @@ TEST_P(XdsServerRdsTest, NonInlineRouteConfigurationNotAvailable) {
                                              backends_[0]->port(),
                                              default_server_route_config_);
   backends_[0]->Start();
-  backends_[0]->notifier()->WaitOnServingStatusChange(
```

Comment:
Wouldn't we want to see non-serving status reported in this case?

Reply:
also, in this case, we continue to serve RPCs, we just fail them.

Reply:
```
If HttpConnectionManager.rds references a NACKed resource without a previous good version, an unavailable resource because of communication failures with control plane or a triggered loading timeout, or a non-existent resource, then all RPCs processed by that HttpConnectionManager will fail with UNAVAILABLE.
```
yup